pkg/kubectl/cmd/util/printing.go

```diff
@@ -30,6 +33,32 @@ func AddPrinterFlags(cmd *cobra.Command) {
 	cmd.Flags().StringP("template", "t", "", "Template string or path to template file to use when -o=template or -o=templatefile.  The template format is golang templates [http://golang.org/pkg/text/template/#pkg-overview]")
 }
 
+// AddOutputFlags adds output related flags to a command. Used by mutations only.
+func AddOutputFlags(cmd *cobra.Command) {
+	cmd.Flags().StringP("output", "o", "", "Output mode. Use -o name for showing only kind/name.")
+}
+
+// PrintSuccess prints message after finishing mutating operations
+func PrintSuccess(cmd *cobra.Command, out io.Writer, resource string, name string, operation string) {
+	outputMode := GetFlagString(cmd, "output")
+	if outputMode == "name" {
+		// -o name: prints kind/name
+		fmt.Fprintf(out, "%s/%s\n", resource, name)
+	} else {
+		// understandable output by default
+		fmt.Fprintf(out, "%s \"%s\" %s successfully\n", resource, name, operation)
```

Comment:
It would be nice to print singular rather than plural here.

It seems like that transformation must exist somewhere -- resource to kind, basically.

RESTMapper sort of has this information, but I don't know that it's usable:
https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/api/meta/restmapper.go#L108

If we can't figure this out, I'm ok with plural for now.